client/signup/steps/plans/index.jsx

```diff
@@ -291,12 +304,11 @@ export class PlansStep extends Component {
 			}
 		}
 
-		return (
-			<>
-				<StepWrapper
+		if ( useStepperWrapper ) {
+			return (
+				<StepperStepContainer
```

Comment:
The header text is missing:

<img width="1202" alt="CleanShot 2024-08-28 at 15 26 51@2x" src="https://github.com/user-attachments/assets/f2e26e8d-1555-4255-b848-64c4b421d951">

We have to use the `headerText` inside of `StepperStepContainer` (it's passed inside the component, but the prop is never used) or pass a `formattedHeader` prop.

Reply:
What URL is this?

Reply:
This was on `/setup/onboarding/plan`. I see that you fixed it though.